input/fsh/Profiles clinical datapoints/TreatmentVariables.fsh

```diff
@@ -0,0 +1,449 @@
+* code = SCT#33496007 "Reconstruction of breast" // found this code myself, is not specifically delayed
+* subject only Reference(BreastCancerPatient)
+* performedDateTime	and complication MS
+* reasonReference only Reference (PrimaryBreastCancerCondition)
+* bodySite MS // need codes for pre pectoral and sub-pectoral reconstruction
+* focalDevice MS // need codes for the different types of impants (direct, staged, autologous implants)
+
+Instance: ReconstructionSurgeryPatient147
+InstanceOf: ReconstructionSurgery
+Description: "Example of a breast cancer patient who underwent reconstruction surgery"
+* status = ProcedureStatusCS#completed
+* code = SCT#33496007 "Reconstruction of breast"
+* subject = Reference(BreastCancerPatient147)
+* performedDateTime = "1999-02-13"
+* reasonReference = Reference(PrimaryBreastCancerPatient147)
+// need codes to create examples of location and type of reconstruction
+
+Mapping: ReconstructionSurgeryToICHOM
+Source:	ReconstructionSurgery
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: reconstruction-surgery-mapping
+Title: "Reconstruction surgery to ICHOM set"
+Description: "Mapping of the reconstruction surgery to the ICHOM breast cancer PCOM set" 
+* -> "Treatment"
+* code -> "Delayed Reconstruction"
+* performedDateTime -> "Surgery date" 
+* bodySite -> "Implant Reconstruction"
+* focalDevice -> "Reconstruction Type"
+
+// RADIOTHERAPY
+Profile: Radiotherapy
+Parent: Procedure 
+Id: radiotherapy
+Title: "Radiotherapy"
+Description: "Represents if the breast cancer patient received radiotherapy during the last year." 
+* code = SCT#108290001 "Radiation oncology AND/OR radiotherapy" 
+* subject only Reference(BreastCancerPatient)
+* performedPeriod MS
+* reasonReference only Reference (PrimaryBreastCancerCondition)
+* reasonCode from TherapyIntentVS (required)
+* bodySite from LocationRadiotherapyVS (preferred) 
+
+Instance: RadiotherapyPatient147
+InstanceOf: Radiotherapy
+Description: "Example of a breast cancer patient who received radiotherapy"
+* status = ProcedureStatusCS#completed
+* code = SCT#108290001 "Radiation oncology AND/OR radiotherapy"
+* subject = Reference(BreastCancerPatient147)
+* performedPeriod.start = "2021-06-19"
+* performedPeriod.end = "2021-12-01"
+* reasonCode = SCT#373846009 "Adjuvant - intent" 
+* reasonReference = Reference(PrimaryBreastCancerPatient147)
+* bodySite = SCT#263601005 "Site of tumor"
+
+Mapping: RadiotherapyToICHOM
+Source:	Radiotherapy
+Target: "https://connect.ichom.org/patient-centered-outcome-measures/breast-cancer"
+Id: radiotherapy-mapping
+Title: "Radiotherapy to ICHOM set"
+Description: "Mapping of radiotherapy to the ICHOM breast cancer PCOM set" 	
+* -> "Treatment"
+* performedPeriod.start -> "Radiotherapy start date" 
+* performedPeriod.end -> "Radiotherapy stop date" 
+* reasonCode -> "Radiotherapy"
+* bodySite -> "Radiotherapy type"
+
+// CHEMOTHERAPY
+Profile: Chemotherapy
+Parent: MedicationAdministration
```

Comment:
For Radiotherapy there is a Procedure profile. Why here only MedicationAdministration? Shouldn't there also be a Procedure linked to MedicationAdministration?